### PR TITLE
Pylint

### PR DESCRIPTION
**Pull request by jspaaks**
In this PR:

- added pylint config from sibling project `howfairis`
- fixed the errors

### CODE REVIEW

File: fairtally/check.py [hunk 1]
```diff
@@ -6,13 +6,15 @@
 
 
 def check_url(url, current_value):
+    """ """
```
Comment (sverhoeven) [inline fairtally/check.py:9]:
```suggestion
    """Run howfairis checks on given url and report current url in tqdm progressbar"""
```

File: fairtally/cli.py [hunk 2]
```diff
@@ -26,6 +26,7 @@
               type=click.Choice(("html", "json")))
 @click.version_option(__version__)
 def cli(urls, input_file, output_format, output_filename):
+    """" """
```
Comment (sverhoeven) [inline fairtally/cli.py:29]:
```suggestion
    """"Command line interface for fairtally"""
```

File: fairtally/redirect_stdout_stderr.py [hunk 3]
```diff
@@ -2,11 +2,14 @@
 
 
 # from https://stackoverflow.com/a/6796752
-class RedirectStdStreams():
+class RedirectStdStreams:
+    """ """
```
Comment (sverhoeven) [inline fairtally/redirect_stdout_stderr.py:6]:
```suggestion
    """Temporary redirect stdout and stderr to given file handles"""
```

File: fairtally/utils.py [hunk 4]
```diff
@@ -6,6 +6,7 @@
 
 
 def merge_urls(urls: Set[str], input_file: TextIO) -> List[str]:
+    """ """
```
Comment (sverhoeven) [inline fairtally/utils.py:9]:
```suggestion
    """Merge URLs given as separate positional arguments and input file"""
```

File: fairtally/utils.py [hunk 5]
```diff
@@ -16,15 +17,17 @@ def merge_urls(urls: Set[str], input_file: TextIO) -> List[str]:
 
 
 def write_as_html(results, output_file_html: TextIO):
+    """ """
```
Comment (sverhoeven) [inline fairtally/utils.py:20]:
```suggestion
    """Writes results to output file in HTML format"""
```

File: fairtally/utils.py [hunk 6]
```diff
@@ -16,15 +17,17 @@ def merge_urls(urls: Set[str], input_file: TextIO) -> List[str]:
...
-        template_str = f.read()
+    with open(template_file) as fid:
+        template_str = fid.read()
     report = template_str.replace("{{python inserts the data here}}", json.dumps(results))
     with output_file_html:
         output_file_html.write(report)
 
 
 def write_as_json(results, output_file_json: TextIO):
+    """ """
```
Comment (sverhoeven) [inline fairtally/utils.py:31]:
```suggestion
    """Write results to output file in JSON format"""
```

**Review by sverhoeven — APPROVED** [review]
Looks good.
Local `pylint fairtally` is not complaining and [superlinter](
https://github.com/fair-software/fairtally/actions/runs/645982730) is also happy.

However, having empty doc strings is a bit of cheat, having proper docstrings would be nicer.
Added suggestions to fill docstrings.

**Comment by jspaaks** [comment]
Thanks for reviewing and the docstrings!

**Pull request merged**